consensus/src/block_storage/sync_manager.rs

```diff
@@ -295,6 +299,7 @@ impl BlockRetriever {
         peers: &mut Vec<&AccountAddress>,
         num_blocks: u64,
     ) -> anyhow::Result<Vec<Block>> {
+        //info!("Retrieving {} blocks starting from {}", num_blocks, block_id);
```

Comment:
why? is it spamming?

Reply:
yes, it spams quite a bit before we changed the limit

Reply:
Yeah a little bit while I was debugging other stuff.